crates/turbopack-cli/src/contexts.rs

```diff
@@ -155,7 +153,6 @@ async fn get_client_module_options_context(
 
     let module_options_context = ModuleOptionsContext {
         enable_jsx,
-        enable_postcss_transform: Some(PostCssTransformOptions::default().cell()),
```

Comment:
Turbopack cli should have PostCSS enabled, so you need to add the postcss rule here too

Reply:
The function to build the postcss rule is currently [in the next.js repo](https://github.com/vercel/next.js/blob/postcss-into-rule/packages/next-swc/crates/next-core/src/next_shared/transforms/postcss.rs#L69). Should we move it to the turbo repo, maybe to [this file](https://github.com/vercel/turbo/blob/postcss-into-rule/crates/turbopack-node/src/transforms/postcss.rs), so we can use it un turbopack-cli?